pydantic/analyzed_type.py

```diff
@@ -0,0 +1,241 @@
+from __future__ import annotations
```

Comment:
```suggestion
from __future__ import annotations as _annotations
```
otherwise it's easy to accidentally use `annotations`.